database/sql/V077__orders_token_indexes.sql

```diff
@@ -0,0 +1,3 @@
+CREATE INDEX orders_sell_buy_tokens ON orders (sell_token, buy_token);
```

Comment:
I think we might have enough indices already to write a fast query for this request.

Reply:
https://github.com/cowprotocol/services/pull/3197#discussion_r1900106270